### DISCUSSION TITLE
How to write PDDL problem that has the objects seperated

### DISCUSSION THREAD

**Discussion by n0whereRuoxi** [aiplan4eu/unified-planning#511] [Q&A]
The objects part looks like:
(:objects
	rover0 - Rover
	waypoint0 waypoint1 waypoint2 - Waypoint
)
How can I use PDDL writer to print something like:
(:objects
	rover0 - Rover
	waypoint0 - Waypoint
        waypoint1 - Waypoint
        waypoint1 - Waypoint
)

**Comment by Framba-Luca** [comment]
Hi @n0whereRuoxi , 

The `PDDLWriter` does not support that, but it should be quite doable to write a script that takes a file written in the first format and prints/dumps the second format.
